internal/constants/constants.go

```diff
@@ -1,7 +1,10 @@
 package constants
 
+//ProjectID GCP project ID
+const ProjectID = "erouska-key-server-dev"
```

Comment:
this should not be hardcoded, I'll make an issue for that and fix it myself:)

Reply:
I know. It's the same as was/is `FirebaseUrl`.